pkg/kernel/networkservice/connectioncontext/ipcontext/server.go

```diff
@@ -137,10 +119,10 @@ func setIPNeighbors(ipNeighbours []*networkservice.IpNeighbor, link netlink.Link
 		}
 		if err := netlink.NeighAdd(&netlink.Neigh{
 			LinkIndex:    link.Attrs().Index,
-			State:        netlink.NUD_REACHABLE,
+			State:        0x02, // netlink.NUD_REACHABLE
```

Comment:
Why not constant?

Reply:
netlink is a bit of linux-specific and all pretty constants such as netlink.FAMILY_ALL, netlink.NUD_REACHABLE are not available for OSx developer :)